modules/quadbin/snowflake/doc/QUADBIN_POLYFILL.md

```diff
@@ -0,0 +1,32 @@
+### QUADBIN_POLYFILL
+
+{{% bannerNote type="code" %}}
+carto.QUADBIN_POLYFILL(geography, resolution)
+{{%/ bannerNote %}}
+
+**Description**
+
+Returns an array of quadbins that intersect with the given geography at a given level of detail.
+
+* `geography`: `GEOGRAPHY` geography to extract the quadbins from.
+* `resolution`: `INT` level of detail or zoom.
+
+**Return type**
+
+`ARRAY<INT>`
+
+{{% customSelector %}}
+**Example**
+{{%/ customSelector %}}
+
+```sql
+SELECT carto.QUADBIN_POLYFILL(
```

Comment:
For this example and the test we are doing the next in other warehouses:
```
carto.QUADBIN_POLYFILL(
    ST_GEOGFROMTEXT('POLYGON ((-3.71219873428345 40.413365349070865, -3.7144088745117 40.40965661286395, -3.70659828186035 40.409525904775634, -3.71219873428345 40.413365349070865))'),
    17);
```

So we avoid all the points, makeline and makepolygon